docs/resolution-getting-started.md

```diff
@@ -24,16 +24,11 @@ with a Pipeline stored in a git repo.
 See [the installation instructions for Tekton Pipeline](./install.md#installing-tekton-pipelines-on-kubernetes), and
 [the installation instructions for the built-in resolvers](./install.md#installing-and-configuring-remote-task-and-pipeline-resolution).
 
-## Step 2: Configure Pipelines to enable alpha features and resolvers
+## Step 2: Configure Pipelines to enable resolvers
 
-Tekton Pipelines currently has its integration with remote resolution behind
-the alpha feature gate, and enabling specific resolvers is controlled by feature 
-flags as well:
+Remote resolvers for Tekton Pipelines are enabled by feature flags:
 
 ```sh
-# update the feature-flags configmap in the tekton-pipelines namespace
-kubectl patch -n tekton-pipelines configmap feature-flags -p '{"data":{"enable-api-fields":"alpha","enable-git-resolver":"true"}}'
-
 # update the resolvers-feature-flags configmap in the tekton-pipelines-resolvers namespace
```

Comment:
wait, do we have two configmaps where users need to enable resolvers? Or was this a docs mistake?

Reply:
The `enable-git-resolver` bit there should have been removed in #5436.